.markdownlint.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
we do not follow "Conventional Commits" in this repo now,
just follow the previous commit message formats